roles/ceph-validate/tasks/check_system.yml

```diff
@@ -55,15 +55,15 @@
 
 - name: fail on unsupported ansible version (1.X)
   fail:
-    msg: "Ansible version must be >= 2.7.x, please update!"
+    msg: "Ansible version must be >= 2.8.x, please update!"
   when: ansible_version.major|int < 2
 
 - name: fail on unsupported ansible version
   fail:
-    msg: "Ansible version must be 2.7 or 2.8!"
+    msg: "Ansible version must be 2.8!"
   when:
     - ansible_version.major|int == 2
-    - ansible_version.minor|int not in [7, 8]
+    - ansible_version.minor|int != 8
```

Comment:
not sure whether we should remove 2.7 from ceph-validate at the moment

Reply:
@fultonj any thoughts on this for RDO ?